runtime/tutor/en/vim-01-beginner.tutor

```diff
@@ -888,12 +907,12 @@ You can find help on just about any subject, by giving an argument to the
 ~~~
 # Lesson 7.2: CREATE A STARTUP SCRIPT
 
-** Enable Vim features. **
+** Enable Neovim features. **
 
-Vim has many more features than Vi, but most of them are disabled by
-default. To start using more features you have to create a "vimrc" file.
+Neovim is a very configurable editor. You can customise it any way you like.
+To start using more features you should create a "init.vim" file.
```

Comment:
```suggestion
To start using more features you should create an "init.vim" file.
```

Reply:
maybe "can"?

Reply:
I don't like can since the previous line is "You can.." so I just removed you should